src/screens/Home/EventList/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback } from "react";
+import { USER_FOOD_DRIVES } from "@app/graphql/queries";
+import { useQuery } from "@apollo/client";
+import { Text } from "react-native";
+import ListItem from "@app/components/ui/ListItem";
+import { format } from "date-fns";
+import styles from "./styles";
+
+const EventList: React.FC = () => {
+  const { loading, error, data } = useQuery(USER_FOOD_DRIVES);
+
+  if (loading) return <Text>Loading...</Text>;
+  if (error) return <Text>Error :(</Text>;
```

Comment:
Add in error message on the component, also use toastProvider to display the error on a snackbar